tests/svinterfaces/run_simple.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
that doesn't seem like something that should be part of this PR

Reply:
Removed; I tend to put small QoL improvements in where I find them if I don't think they warrant a PR of their own.